lib/orbf/rules_engine/builders/calculator_factory.rb

```diff
@@ -88,6 +88,13 @@ def self.build(options = { nested_data_support: false, case_sensitive: true })
           calculator.add_function(:between, :logical, BETWEEN)
           calculator.add_function(:abs, :number, ->(number) { number.abs })
           calculator.add_function(:score_table, :numeric, SCORE_TABLE)
+          # Dentaku now has AVG and SUM builtin, as well as ROUNDDOWN and ROUNDUP
+          # Their behavior is slightly different.
+          #      AVG(1,3,9) => In our function: 4,333333
+          #                    In Dentaku: 4.0 (if one of the elements is a float, it does work the same)
+          #
+          # Since we're mostly using our go-hesabu library, I'm just keeping these as is and ignoring the
+          # builtin functions.
```

Comment:
If I understand well,  our functions still "override" theirs ?

Reply:
Yes, we're "winning"